aerogear/mobile-security-service-operator
# fix(makefile): fixed make test command (AEROGEAR-9238)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-9238

## What

Running `make test` gives this bash warning:

```sh
$ make test
Running tests:
/bin/sh: line 0: [: too many arguments
```

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Run `make test`
2. The bash warning should not appear. You should see:

```sh
$ make test
Running tests:
...
```

## Checklist:


## Progress
